Kernel/Syscalls/select.cpp

```diff
@@ -48,11 +48,16 @@ KResultOr<int> Process::sys$select(Userspace<const Syscall::SC_select_params*> u
     });
 
     fd_set fds_read, fds_write, fds_except;
-    if (params.readfds && !copy_from_user(&fds_read, params.readfds))
+
+    size_t bytes_used = (params.nfds / 8) + ((params.nfds % 8) == 0 ? 0 : 1);
```

Comment:
You can use `ceil_div()` (from `AK/StdLibExtras.h`) to make this sort of thing more readable. :)

Reply:
Cool, fixed. :+1: